WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -257,7 +256,7 @@ public void onLog(T tag, LogLevel logLevel, String message) {
                 StringBuffer sb = new StringBuffer();
                 sb.append(logLevel.toString()).append("/").append(AppLog.TAG).append("-")
                   .append(tag.toString()).append(": ").append(message);
-                CrashLoggingUtils.log(sb.toString());
+                mCrashLogging.log(sb.toString());
```

Comment:
We still log every log entry to breadcrumbs until encrypted logging ships – then IMHO we should remove this.

Reply:
I agree. We might consider keeping barebones of information here, but we'll likely just remove the whole thing.